services/QuillLMS/spec/models/school_spec.rb

```diff
@@ -160,4 +164,26 @@
       expect(School.school_year_start(time)).to eq(Date.parse('2019-07-01').beginning_of_day)
     end
   end
+
+  describe('district admin behavior') do
+    let(:school) { create(:school) }
+    let(:district) { create(:district) }
+    let(:admin) { create(:user)}
+
+    it 'creates new school admin record if a school is attached to a new district' do
```

Comment:
Again may be worth a test case for what happens if the admin is already attached to the school (manually or whatever) when this happens.